vtr_flow/scripts/python_libs/vtr/flow.py

```diff
@@ -0,0 +1,288 @@
+import sys
+import shutil
+import subprocess
+import time
+import glob
+import vtr
+from pathlib import Path
+from collections import OrderedDict
+from vtr import CommandError
+
+VTR_STAGE = vtr.make_enum("odin", "abc", 'ace', "vpr")
```

Comment:
Python has an enum class now.  Can we just use that instead?

Reply:
The python enum has now been implemented.